recipe/meta.yaml

```diff
@@ -14,10 +14,11 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Build number bump is not enough. You need a version number bump

Reply:
We can mark build 0 broken and I think we'll be fine.

Reply:
We also need to mark any v4 pybind11 Abi packages as broken too.

Reply:
Wouldn't it be better to bump version to v5 and rebuild?

Reply:
The pybind11-abi version is supposed to track the pybind11 internals version. So we'd make them out of sync. We could append a minor version to it that is the conda-forge one. This has the risk of being an issue it pybind11 changes its scheme later. We could also preprend a 0. to it which avoids the upstream changes.

Reply:
We could also just let conda-forge have an extra version increment.

Reply:
See my latest commit for a possible fix.

Reply:
Not a fan of that. Why not use a `epoch`?

Reply:
We can go to epochs. They are ugly and I do not like them but I don't have strong opinions here.

Reply:
Hopefully mamba can handle them?

Reply:
OK. epochs pushed.